packages/lib/server/getServerErrorFromUnknown.ts

```diff
@@ -56,8 +55,8 @@ export function getServerErrorFromUnknown(cause: unknown): HttpError {
   if (isPrismaError(cause)) {
     return getServerErrorFromPrismaError(cause);
   }
-  if (cause instanceof Stripe.errors.StripeInvalidRequestError) {
-    return getHttpError({ statusCode: 400, cause });
+  if ((cause as any)?.type === "StripeInvalidRequestError") {
+    return getHttpError({ statusCode: 400, cause: cause as any });
```

Comment:
<img width="558" alt="Screenshot 2025-02-26 at 10 48 27 AM" src="https://github.com/user-attachments/assets/36d94b2f-f38d-480d-b404-548b6f236ed2" />

Reply:
Let's copy and paste the relevant parts of the source code instead of relying on "any"s